frameworks/remix-run/application/src/routes/$langstore_.$.tsx

```diff
@@ -49,7 +49,7 @@ export const loader: LoaderFunction = async ({ request, params }) => {
 };
 
 export default () => {
-    const { data, shapeIdentifier } = useLoaderData();
+    const { data, shapeIdentifier } = useLoaderData() as any;
```

Comment:
`useLoaderData<typeof loader>();` + change the loader using `LoaderFunctionArgs`